src/common/evaluation/QA/eval_depthmap_models/src/utils.py

```diff
@@ -108,10 +111,10 @@ def calculate_and_save_results(MAE, complete_name, CSV_OUT_PATH, DATA_CONFIG, RE
     result = pd.concat(dfs)
     result.index.name = 'Model_Scantype'
     result = result.round(2)
-    print(result)
-
     # Save the model results in csv file
-    result.to_csv(CSV_OUT_PATH, index=True)
+    pathlib.Path(CSV_OUT_PATH).mkdir(parents=True, exist_ok=True)
+    csv_file = "{}{}{}{}".format(CSV_OUT_PATH, '/', RUN_ID, '.csv')
```

Comment:
Modern format strings are more readable 
```suggestion
    csv_file = f"{CSV_OUT_PATH}/{RUN_ID}.csv"
```